docs/core/linux-prerequisites.md

```diff
@@ -20,25 +20,29 @@ This article shows the dependencies needed to develop .NET Core applications on
 * [Command-line with your favorite editor](tutorials/using-with-xplat-cli.md)
 * [Visual Studio Code](https://code.visualstudio.com/)
 
+> [!NOTE]
+> The .NET Core SDK (.dotnet-sdk-x-x64.x.x.x) package is not required for production servers/environments. Only the .NET Core runtime package (.dotnet-x-x64.x.x.x) is needed for apps deployed to production environments. The .NET Core runtime is deployed with apps as part of a self-contained deployment, however, it must be deployed for Framework-dependent deployed apps seperately. For general information about Framework-dependent and Self-contained deployment types, please see [.NET Core application deployment](https://docs.microsoft.com/en-us/dotnet/core/deploying/). Also see [Self-contained Linux applications](https://github.com/dotnet/core/blob/master/Documentation/self-contained-linux-apps.md) for specific guidelines.
+
 ## Supported Linux versions
 
 # [.NET Core 2.x](#tab/netcore2x)
 
-.NET Core 2.0 treats Linux as a single operating system. There is a single Linux build (per chip architecture) for supported Linux distros.
+.NET Core 2.x treats Linux as a single operating system. There is a single Linux build (per chip architecture) for supported Linux distributions.
 
 NET Core 2.x is supported on the following Linux 64-bit (`x86_64` or `amd64`) distributions/versions:
 
- * Red Hat Enterprise Linux 7
- * CentOS 7
- * Oracle Linux 7
- * Fedora 25, Fedora 26
- * Debian 8.7 or later versions 
- * Ubuntu 17.04, Ubuntu 16.04, Ubuntu 14.04
- * Linux Mint 18, Linux Mint 17
- * openSUSE 42.2 or later versions
- * SUSE Enterprise Linux (SLES) 12 SP2 or later versions
+* Red Hat Enterprise Linux 7
+* CentOS 7
+* Oracle Linux 7
+* Fedora 27, 26
+* Debian 9, 8.7 or later versions
+* Ubuntu 17.10, 16.04, 14.04
```

Comment:
Ubuntu 18.04 will be released this month. We may as well add it to the list now.

Reply:
@aaronfranke, as soon as Ubuntu 18.04 is supported, we'll be sure to add it. Thanks!